sonar-plugin-api/src/main/java/org/sonar/api/batch/DefaultFormulaContext.java

```diff
@@ -19,37 +19,42 @@
  */
 package org.sonar.api.batch;
 
+import org.sonar.api.measures.Formula;
 import org.sonar.api.measures.FormulaContext;
 import org.sonar.api.measures.Metric;
 import org.sonar.api.resources.Resource;
 
 /**
  * @since 1.11
+ * @deprecated since 5.2. Aggregation of measures is provided by {@link org.sonar.api.ce.measure.MeasureComputer}. {@link org.sonar.api.batch.Decorator}
+ * and {@link Formula} are no more supported.
  */
+@Deprecated
 public class DefaultFormulaContext implements FormulaContext {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)